browser/css/jssidebar.css

```diff
@@ -353,6 +353,10 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 	margin-block-end: 8px;
 }
 
+#NavigatorPanel .ui-treeview-entry.highlighted {
+	background-color: orange;
```

Comment:
possibly we want some variable here, like `var(--primary-color)`
but primary is used for "currently active" entry so we will need different one